app/create_db_tables.py

```diff
@@ -20,12 +20,28 @@ def initialize_police_table():
         title TEXT,
         description TEXT,
         tags TEXT,
-        force_rank TEXT
+        force_rank TEXT,
+        confidence FLOAT
+    )""",
+
```

Comment:
Is this table title clear?

Reply:
We'll leave it as in_process for now, definitely worth changing if we come up with a better name.